### PR TITLE
Treat "unsure" as both "yes" and "no"

### PR DESCRIPTION
**Pull request by NickCrews**
closes https://github.com/dedupeio/dedupe/issues/984

### CODE REVIEW

File: dedupe/convenience.py [hunk 1]
```diff
@@ -88,101 +88,101 @@ def randomPairsWithReplacement(n_records: int, sample_size: int) -> IndicesItera
...
     buffer_len = 1  # Max number of previous operations
-    examples_buffer: List[Tuple[TrainingExample, Literal['match', 'distinct', 'uncertain']]] = []
-    uncertain_pairs: List[TrainingExample] = []
+    unlabeled: List[TrainingExample] = []
+    labeled: List[LabeledPair] = []
 
     while not finished:
         if use_previous:
-            record_pair, _ = examples_buffer.pop(0)
+            record_pair, _ = labeled.pop(0)
```
Comment (fgregg) [inline dedupe/convenience.py:127]: we would need to pop twice in the case of unsure, right?

**Review by fgregg — CHANGES_REQUESTED** [review]

File: dedupe/convenience.py [hunk 2]
```diff
@@ -88,101 +88,101 @@ def randomPairsWithReplacement(n_records: int, sample_size: int) -> IndicesItera
...
-                examples: TrainingData
-                examples = {'distinct': [],
-                            'match': []}
-                examples[label].append(record_pair)
-                deduper.mark_pairs(examples)
+            unlabeled.append(record_pair)
 
-    for record_pair, label in examples_buffer:
-        if label in ['distinct', 'match']:
+        while len(labeled) > buffer_len:
```
Comment (fgregg) [inline dedupe/convenience.py:181]: we'll also need to do this a bit differently, since unsure adds to elements to labeled

**Review by fgregg — CHANGES_REQUESTED** [review]

File: dedupe/convenience.py (hunk 1 above)
Comment (NickCrews) [reply dedupe/convenience.py:133]: Oops, yeah. See fixup commit

**Review by fgregg — APPROVED** [review]

**Comment by fgregg** [comment]
thanks, @NickCrews !

**Pull request merged**